asv_bench/benchmarks/binary_ops.py

```diff
@@ -20,94 +23,87 @@ def setup(self, use_numexpr, threads):
         if not use_numexpr:
             expr.set_use_numexpr(False)
 
-
     def time_frame_add(self, use_numexpr, threads):
-        (self.df + self.df2)
+        self.df + self.df2
 
     def time_frame_mult(self, use_numexpr, threads):
-        (self.df * self.df2)
+        self.df * self.df2
 
     def time_frame_multi_and(self, use_numexpr, threads):
-        self.df[((self.df > 0) & (self.df2 > 0))]
+        self.df[(self.df > 0) & (self.df2 > 0)]
 
     def time_frame_comparison(self, use_numexpr, threads):
-        (self.df > self.df2)
+        self.df > self.df2
 
     def teardown(self, use_numexpr, threads):
         expr.set_use_numexpr(True)
         expr.set_numexpr_threads()
 
 
 class Ops2(object):
+
     goal_time = 0.2
 
     def setup(self):
-        self.df = DataFrame(np.random.randn(1000, 1000))
-        self.df2 = DataFrame(np.random.randn(1000, 1000))
+        N = 10**3
+        np.random.seed(1234)
+        self.df = DataFrame(np.random.randn(N, N))
+        self.df2 = DataFrame(np.random.randn(N, N))
 
-        self.df_int = DataFrame(
-            np.random.random_integers(np.iinfo(np.int16).min,
-                                      np.iinfo(np.int16).max,
-                                      size=(1000, 1000)))
-        self.df2_int = DataFrame(
-            np.random.random_integers(np.iinfo(np.int16).min,
-                                      np.iinfo(np.int16).max,
-                                      size=(1000, 1000)))
+        self.df_int = DataFrame(np.random.randint(np.iinfo(np.int16).min,
+                                                  np.iinfo(np.int16).max,
+                                                  size=(N, N)))
+        self.df2_int = DataFrame(np.random.randint(np.iinfo(np.int16).min,
+                                                   np.iinfo(np.int16).max,
+                                                   size=(N, N)))
 
-    ## Division
+    # Division
 
     def time_frame_float_div(self):
-        (self.df // self.df2)
+        self.df // self.df2
 
     def time_frame_float_div_by_zero(self):
-        (self.df / 0)
+        self.df / 0
 
     def time_frame_float_floor_by_zero(self):
-        (self.df // 0)
+        self.df // 0
 
     def time_frame_int_div_by_zero(self):
-        (self.df_int / 0)
+        self.df_int / 0
 
-    ## Modulo
+    # Modulo
 
     def time_frame_int_mod(self):
-        (self.df / self.df2)
+        self.df_int % self.df2_int
 
     def time_frame_float_mod(self):
-        (self.df / self.df2)
+        self.df % self.df2
 
 
 class Timeseries(object):
+
     goal_time = 0.2
 
-    def setup(self):
-        self.N = 1000000
+    params = [None, 'US/Eastern']
```

Comment:
http://asv.readthedocs.io/en/latest/writing_benchmarks.html

we can add ``params_names`` (here) and elsewhere (just for readibility)